services/meta/client_test.go

```diff
@@ -108,30 +110,33 @@ func TestMetaClient_CreateDatabaseWithRetentionPolicy(t *testing.T) {
 
 	// Recreating the exact same database with retention policy is not
 	// an error.
-	if _, err := c.CreateDatabaseWithRetentionPolicy("db0", &rpi); err != nil {
+	if _, err := c.CreateDatabaseWithRetentionPolicy("db0", &spec); err != nil {
 		t.Fatal(err)
 	}
 
 	// If the rp's duration is different, an error should be returned.
-	rpi2 := rpi
-	rpi2.Duration = rpi.Duration + time.Minute
-	if _, err := c.CreateDatabaseWithRetentionPolicy("db0", &rpi2); err != meta.ErrRetentionPolicyConflict {
-		t.Fatalf("got %v, but expected %v", err, meta.ErrRetentionPolicyConflict)
-	}
-
-	// If the rp's replica is different, an error should be returned.
-	rpi2 = rpi
-	rpi2.ReplicaN = rpi.ReplicaN + 1
-	if _, err := c.CreateDatabaseWithRetentionPolicy("db0", &rpi2); err != meta.ErrRetentionPolicyConflict {
+	spec2 := spec
+	duration2 := *spec.Duration + time.Minute
+	spec2.Duration = &duration2
+	if _, err := c.CreateDatabaseWithRetentionPolicy("db0", &spec2); err != meta.ErrRetentionPolicyConflict {
 		t.Fatalf("got %v, but expected %v", err, meta.ErrRetentionPolicyConflict)
 	}
 
-	// If the rp's shard group duration is different, an error should be returned.
-	rpi2 = rpi
-	rpi2.ShardGroupDuration = rpi.ShardGroupDuration + time.Minute
-	if _, err := c.CreateDatabaseWithRetentionPolicy("db0", &rpi2); err != meta.ErrRetentionPolicyConflict {
-		t.Fatalf("got %v, but expected %v", err, meta.ErrRetentionPolicyConflict)
-	}
+	/*
```

Comment:
Nope. It looks like I got distracted while working on this and forgot about the comment.